packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+
+abstract class BaseInfo<T> {
```

Comment:
I thought of using *EventInfo but since this name will be repeated a lot just Info might be clear enough and save a lot of typing. I think javadocs here is enough

Reply:
This inheritance actually, is only useful for DRY, I am actually make it private.

Reply:
I thought it could be useful for defining generic handlers?